src/site/topic23.rst

```diff
@@ -332,10 +332,21 @@ General Remove
 Contains
 --------
 
+* Like the other data structures, we will want to be able to check if a given element exists within it
+* Unlike before, where we were doing a linear search through a linear data structure
+* Or an exhaustive depth first search through a regular binary tree
```

Comment:
_or..._ meant to be indented?

Reply:
It wasn't, but I could see it going both ways

1. linear
2. non linear

3. Fancy nonlinear